docs/windows_faq.md

```diff
@@ -0,0 +1,26 @@
+When running on windows there are a few quick tips that we recommend to help
+your file I/O performance.
+
+## Exclude your package directory from Windows Defender
+
+The build system does a lot of file I/O, especially when using the `-o` option
+or running tests.
+
+You can exclude your package entirely from windows defender by following
```

Comment:
Could call out what the risks are - that external dependency source files are _not_ included anywhere in the package directory, but that external dependency sources _can_ generate files inside `.dart_tool/build` when running a build.

Reply:
done